.dockerignore

```diff
@@ -1 +1,8 @@
-*.lock
+.tox/*
```

Comment:
That could have unintended consequences as it would exclude more files from the docker context than are ignored by .gitignore. I doubt that anyone would ever hit that issue, but it would be probably impossible to find.